### ISSUE TITLE
problem with install pod

### ISSUE THREAD

**Issue by ol3j** [mCodex/react-native-sensitive-info#137]
Hi, 

I have problem with install pod for ios. 
I run 
* yarn add react-native-sensitive-info
* react-native link react-native-sensitive-info
next I go to folder ios and run
pod install

I have the message.  

```
Fetching podspec for `react-native-sensitive-info` from `../node_modules/react-native-sensitive-info`
[!] CocoaPods could not find compatible versions for pod "react-native-sensitive-info":
  In Podfile:
    react-native-sensitive-info (from `../node_modules/react-native-sensitive-info`)

Specs satisfying the `react-native-sensitive-info (from `../node_modules/react-native-sensitive-info`)` dependency were found, but they required a higher minimum deployment target.
```
What is wrong?

**Comment by BenjErgizerBunny** [comment]
I had the same issue when trying to run pod install after doing react-native link because when only doing react-native link I get an error when calling SInfo.setItem that SInfo is not defined

**Comment by khorark** [comment]
Hi! React-native-sensitive-info need min target ios version 9.1 . Please, open your Podfile and check ios version. Should be "platform :ios, '9.1'" or hight.

**Comment by savelichalex** [comment]
Could someone told why version 9.1 is required?

**Comment by dcalhoun** [comment]
@savelichalex there is [no reason for the 9.1 requirement](https://github.com/mCodex/react-native-sensitive-info/issues/46#issuecomment-332289521). It was [rolled back 8.0](https://github.com/mCodex/react-native-sensitive-info/pull/48). However, there appears to have been an oversight and the [podspec file was not correctly rolled back](https://github.com/mCodex/react-native-sensitive-info/blob/06bfb79fe14b8d0950084f5870cfebb3b59ba4ca/react-native-sensitive-info.podspec#L9).

I will try to open a PR soon to fix this issue.

**Comment by Abramovick** [comment]
Was this ever fixed, I'm still getting that same error.

**Comment by savelichalex** [comment]
@Abramovick it's still here https://github.com/mCodex/react-native-sensitive-info/blob/master/react-native-sensitive-info.podspec#L9

**Comment by Abramovick** [comment]
@savelichalex thanks bro.
It a really really simple fix, not sure why it's not been changed since June...

**Comment by savelichalex** [comment]
@Abramovick yeah, I know it's frustrating... Anyway, you can create your own copy of this podspec and place it somewhere in your folder, then just use it instead default one, as a quick temporal fix.

**Comment by mCodex** [comment]
I rolled back .podspec to 8.0, can you see if 5.5.1 works for you?

**Comment by digitaluniverse** [comment]
[](https://github.com/pradeep1991singh/react-native-secure-key-store) was my way of fixing this issue

**Comment by arnoldc** [comment]
For the keystore branch, its not yet fix
it still says  
s.platform      = :ios, "9.1 

see code
https://github.com/mCodex/react-native-sensitive-info/blob/keystore/react-native-sensitive-info.podspec

**Comment by mCodex** [comment]
> For the keystore branch, its not yet fix
> it still says
> s.platform = :ios, "9.1
> 
> see code
> https://github.com/mCodex/react-native-sensitive-info/blob/keystore/react-native-sensitive-info.podspec

Done.
